.appveyor.yml

```diff
@@ -71,6 +71,9 @@ install:
   - flutter config --no-analytics
   - flutter doctor
   - pip install pipenv==%PYTHON_PIPENV_VERSION%
+  # Install Conda
+  - powershell .\\utils\\conda_install.ps1
```

Comment:
Please call the powershell using `ps: ` like above.
Where is this file supposed to come from? Don't you need to download conda first?

Reply:
I added the file to repository including the license header from [reference in the conda docs](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#appveyor).